dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/user/UserControllerTest.java

```diff
@@ -130,9 +129,9 @@ public void setUp()
         parsedUser.setGroups( new HashSet<>( Arrays.asList( userGroup1, userGroup2 ) ) );
     }
 
+    // @Test//Fails12577
```

Comment:
use `@Ignore` on failing test, don't comment away the `@Test`. This also gives you a way to leave a comment as argument to the `@Ignore` so is clear why it was put there and what it related to and when it can be removed again.

Reply:
I just found out that we must use @Disabled instead of @Ignore. It might be related to the latest junit upgrade.